src/server/game/Entities/Player/Player.cpp

```diff
@@ -6609,6 +6609,20 @@ bool Player::HasCurrency(uint32 id, uint32 count) const
     return itr != _currencyStorage.end() && itr->second.Quantity >= count;
 }
 
+bool Player::IsQuestObjectiveProgressComplete(Quest const* quest) const
+{
+	float progress = 0;
+	for (QuestObjective const& obj : quest->GetObjectives())
+	{
+		if (obj.Flags & 64) {
```

Comment:
magic number 64 + starting curly brace on new line + fix indentation in all diff lines.